doc/source/whatsnew/v0.17.0.txt

```diff
@@ -45,6 +45,45 @@ Other API Changes
 - Enable writing Excel files in :ref:`memory <_io.excel_writing_buffer>` using StringIO/BytesIO (:issue:`7074`)
 - Enable serialization of lists and dicts to strings in ExcelWriter (:issue:`8188`)
 
+- ``DataFrame.convert_objects`` keyword arguments have been shortened. (:issue:`10265`)
+
+=====================   =============
+Old                     New
+=====================   =============
+``convert_dates``       ``datetime``
+``convert_numeric``     ``numeric``
+``convert_timedelta``   ``timedelta``
+=====================   =============
+
+- Coercing types with ``DataFrame.convert_objects`` is now implemented using the
+keyword argument ``coerce=True``.  Previously types were coerced by setting a
+keyword argument to ``'coerce'`` instead of ``True``, as in ``convert_dates='coerce'``.
+
+  Previously data was coerced to a common dtype before serialisation, which for
+  example resulted in integers being serialised to floats:
```

Comment:
I don't understand this: when I run the example below with 0.16.2, I get integers:

```
In [19]: pd.__version__
Out[19]: '0.16.2'

In [20]: df = pd.DataFrame({'i': ['1','2'], 'f': ['apple', '4.2']})

In [21]: df.convert_objects(convert_numeric='coerce')
Out[21]:
     f  i
0  NaN  1
1  4.2  2

In [22]: df.convert_objects(convert_numeric='coerce').dtypes
Out[22]:
f    float64
i      int64
dtype: object
```